SIM/RetrievalManagement.md

```diff
@@ -0,0 +1,230 @@
+# Retrieval Management Requirements and design
+
+
+
+**Author(s)**: [Pravin Ranjan](https://github.com/pravinranjan10)
+
+Major Version Updates
+
+Date 13/07/2020 : Version : Description  : Author
+
+
+
+## Goals
+* Retrieve or collect the resources(storage, volumes and pools)
+  information from registered backends.
+* Perform the syncing process of resources whenever required and/or
+  periodically.
+
+
+## Motivation and background
+Delfin is an infrastructure management platform for heterogeneous storage environments in SODA foundation. To support this project, resources discovery and data collection required which are implemented in this module.   
+
+### Non-Goals
+
+* Currently periodic Synchronization is not considered
+
+### Assumptions and Constraints
+
+NA
+
+### Requirement Analysis
+
+### Input Requirements
+
+Overall Input requirements can be referred from [here](https://github.com/sodafoundation/design-specs/blob/master/specs/SIM/SODA_InfrastructureManagerDesign.md)
+
+
+### Feature Requirements
+
+#### Requirement Analysis
+
+There are few major requirements for this module:
+1. Develop framework for task manager
+2. Collect Pool  information
+3. Collect LUN(Volume) information
+4. Collect device information
```

Comment:
Other requirements like "task scheduling" , "synchronization" , "performance statistics collection" etc can be added

Reply:
Done